Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -177,7 +192,7 @@ extension SwiftPackageRegistryTool {
                     packageIdentity: self.packageIdentity,
                     packageVersion: self.packageVersion,
                     packageArchive: archivePath,
-                    packageMetadata: publishRequirements.metadata.location.contains(.request) ? metadataPath : .none,
+                    packageMetadata: publishConfiguration.metadata.location.contains(.request) ? metadataPath : .none,
```

Comment:
This can be `self.customMetadataPath` I think

Reply:
this behavior has not been updated yet to the logic mentioned above, it is based on the PublishConfiguration (which used to be set by the server)